packages/config/config/devices/0x027a/zen17.json

```diff
@@ -66,7 +66,7 @@
 		},
 		{
 			"#": "2",
-			"$if": "firmwareVersion >= 1.4",
+			"$if": "firmwareVersion >= 1.1",
```

Comment:
I didn't find this info, can you tell me where you did?

Reply:
I emailed Zooz asking for the latest firmware. They said the newest file is 1.2. On their advanced parameters page it states that all of those apply from firmware 1.1. The garage mode is shown on those instructions, so anything newer than 1.1 should work.

Reply:
Ok just to be sure: This is actually firmware 1.1 and 1.2 and not 1.10 and 1.20?
Because the website says this:
> Here is a full list of advanced settings (parameters) for the ZEN17 Universal Relay VER. **1.10** firmware:

and I know they have been numbering their firmware version weirdly (i.e. only multiples of 10) for display purposes.

Reply:
I asked them about that… this is what they said:

“Home Assistant omits the 0 when displaying firmware files, which means firmware 1.02 displays as 1.2, as does 1.20.“

So it seems that the latest firmware is actually 1.02 but HA reports it as 1.20. What is then the right way of setting this up in the code?

Reply:
> which means firmware 1.02 displays as 1.2, as does 1.20

Yeah, thats not true. 1.02 displays as 1.2, 1.20 as 1.20.

In Z-Wave, the versions are encoded just two numbers, so what they call `1.02` is **major version 1, minor version 2**. Both can go up to 255, so if we were to pad them with leading zeroes, the "correct" way to display them would be `1.002`. This looks awkward and would also not be in line with what some manufacturers (including Zooz) use. 

In fact, we use [semver](https://semver.org/) to compare versions, except we omit the patch part.
> A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes. X is the major version, Y is the minor version, and Z is the patch version. Each element MUST increase numerically. For instance: 1.9.0 -> 1.10.0 -> 1.11.0.

So Z-Wave JS displays the version numbers as-is, with a `.` in the middle, which is the least ambiguous IMO, especially if you're used to software development.

Reply:
So to reiterate: the documented version is `1.10` and what they sent you is `1.20` as I suspected.

Reply:
Got it. Thanks for the thorough explanation.

Do I need to change the firmware to 1.10 or is 1.1 good? Let me know if anything else is missing.